### ISSUE TITLE
game crash on boot

### ISSUE THREAD

**Issue by skezers** [Globox1997/EnvironmentZ#110]
### Minecraft Version

1.20.x

### Mod Loader Version

0.14.24

### Mod Version

2.0.7

### Describe the Issue

Hello, sorry to bother, but the game crash when i launch it with EnvironmentZ 

### latest.log

[crash-2023-11-14_23.51.29-client.txt](https://github.com/Globox1997/EnvironmentZ/files/13358082/crash-2023-11-14_23.51.29-client.txt)

### Do you use any other mods except the required ones?

i tried with all my mods disabled and it still didn't work

**Comment by ImNotAnArtist** [comment]
Yeah same here, tried with required mods and even by its self. Same crash report too

**Comment by Globox1997** [comment]
Hopefully fixed with the latest update.

**Comment by Globox1997** [comment]
Should be fixed.
